.chloggen/semconv-v1.27.0.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: semconv
+note: Add semantic conventions version v1.27.0
+issues: [  ]
```

Comment:
```suggestion
issues: [10837]
```

Reply:
couldn't commit this change directly here, ended up creating a PR against the branch instead